pkg/collector/corechecks/snmp/common/contants.go

```diff
@@ -13,3 +13,23 @@ const SnmpIntegrationName = "snmp"
 
 // SnmpExternalTagsSourceType is the source id used for external tags
 const SnmpExternalTagsSourceType = "snmp"
+
+type IfAdminStatus int
+
+const (
+	AdminStatus_Up IfAdminStatus = iota + 1
+	AdminStatus_Down
+	AdminStatus_Testing
+)
+
+type IfOperStatus int
+
+const (
+	OperStatus_Up IfOperStatus = iota + 1
+	OperStatus_Down
+	OperStatus_Testing
+	OperStatus_Unknown
+	OperStatus_Dormant
+	OperStatus_NotPresent
+	OperStatus_LowerLayerDown
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I think explicitly setting constant could be more readable than using `iota`.

I think iota is great when we just want each constant/enum to have a specific value, we don't care about the value and the value is not explicitly defined somewhere. (But in this case, the value is explicitly defined in IF-MIB and it's easier to match/read if each status matches it's value)